cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -226,3 +227,49 @@ message SegYSeismicResponse {
 message SearchFilesRequest {
     SearchSpec spec = 1;
 }
+
+message SearchJobStatusRequest {
+    message Filter {
+        message Id {
+            oneof id {
+                string job_id = 1;
+                string file_id = 2;
+                string file_name = 3;
+            }
+        }
+
+        oneof filter {
+            Id id = 1;
+            JobStatus status = 2;
+            string target_storage_tier_name = 4;
+            google.protobuf.Timestamp started_before = 5;
+            google.protobuf.Timestamp started_after = 6;
+            google.protobuf.Timestamp updated_before = 7;
+            google.protobuf.Timestamp updated_after = 8;
+        }
+    }
+
+    enum Order {
+        DESCENDING = 0;
+        ASCENDING = 1;
+    }
+
+    Order order_by_updated = 1; // Defaults to most recently updated first
+    repeated Filter filters = 2; // Multiple filters further restrict returned values
+}
+
+message JobStatusResponse {
+    string job_id = 1;
+    string file_id = 2;
+    string file_name = 3;
+    JobStatus status = 4;
+    google.protobuf.Timestamp started_at = 5;
+    google.protobuf.Timestamp updated_at = 6;
+    string target_storage_tier_name = 7;
+    repeated IngestionLog logs = 8;
```

Comment:
I removed the `message` field that exists in the v0 job status response, since the message will just be included in logs anyways, but maybe users would still prefer that message field to exist

Reply:
Hmmm. I think the PGS folks are migrating or have migrated, so given that we have extensive logs I agree that we can probably remove the message field.